postgres/src/test/scala/zio/sql/postgresql/ShopSchema.scala

```diff
@@ -7,10 +7,12 @@ trait ShopSchema extends Jdbc { self =>
 
   object Customers     {
     val customers =
-      (uuid("id") ++ localDate("dob") ++ string("first_name") ++ string("last_name") ++ boolean("verified"))
+      (uuid("id") ++ localDate("dob") ++ string("first_name") ++ string("last_name") ++ boolean(
+        "verified"
+      ) ++ zonedDateTime("created_timestamp"))
```

Comment:
I think the approach you were starting here looks better than adding a separate table